plugins/TelescopeControl/src/gui/TelescopeDialog.cpp

```diff
@@ -260,9 +263,10 @@ void TelescopeDialog::setAboutText()
 	aboutPage += "<h2>" + q_("Telescope Control plug-in") + "</h2><table width=\"90%\">";
 	aboutPage += "<tr width=\"30%\"><td><strong>" + q_("Version") + ":</strong></td><td>" + TELESCOPE_CONTROL_PLUGIN_VERSION + "</td></tr>";
 	aboutPage += "<tr><td><strong>" + q_("License") + ":</strong></td><td>" + TELESCOPE_CONTROL_PLUGIN_LICENSE + "</td></tr>";
-	aboutPage += "<tr><td rowspan=3><strong>" + q_("Authors") + "</strong></td><td>Johannes Gajdosik</td></td>";
+	aboutPage += "<tr><td rowspan=4><strong>" + q_("Authors") + "</strong></td><td>Johannes Gajdosik</td></td>";
 	aboutPage += "<tr><td>Michael Heinz</td></tr>";
 	aboutPage += "<tr><td>Bogdan Marinov &lt;bogdan.marinov84@gmail.com&gt; (" + q_("Plug-in and GUI programming") + ")</td></tr>";
+	aboutPage += "<tr><td>Gion Kunz &lt;gion.kunz@gmail.com&gt; (" + q_("ASCOM Telescope Client") + ")</td></tr>";
```

Comment:
If we still discern between authors (original plugin designers) and contributors (later modifiers): Given that Petr added RTS2, Alessandro added INDI, this should make you a "contributor" of ASCOM support, please.